app/controllers/user_controller.py

```diff
@@ -7,8 +7,8 @@
 
 from app.configs.database import db
 from app.exceptions.city_exc import (
-    CityNotFoundError,
     CityOutOfRangeError,
+    InvalidZipCodeError,
     ZipCodeNotFoundError,
```

Comment:
@JOSUE-ASV  is working in a task that will replace most of this errors

Reply:
By now it's ok.